openedx/core/djangoapps/content/course_overviews/tests.py

```diff
@@ -470,12 +470,18 @@ def test_get_all_courses_by_org(self):
                 for __ in range(3)
             ])
 
-        self.assertSetEqual(
+        self.assertEqual(
```

Comment:
I think it's cleaner. The `assertEqual()` method takes care of dispatching equality checks for objects of the same type to the correct type-specific methods (in this case, `assertSetEqual()`). For more, see the [unittest docs](https://docs.python.org/2/library/unittest.html#unittest.TestCase.addTypeEqualityFunc).